Makefile

```diff
@@ -189,11 +189,6 @@ update-beats: update-beats-module update
 update-beats-module:
 	$(GO) get -d $(BEATS_MODULE)@$(BEATS_VERSION) && $(GO) mod tidy
 
-.PHONY: update-beats-docs
-update-beats-docs:
```

Comment:
question: why this change? Does not seem related to the other change or with the linked issue to me, but I may lack the context. Could you clarify it?